src/language/CodeInspection.js

```diff
@@ -272,7 +272,7 @@ define(function (require, exports, module) {
      * @param {Array.<{name:string, scanFileAsync:?function(string, string):!{$.Promise}, scanFile:?function(string, string):Object}>} providersReportingProblems - providers that reported problems
      * @param {boolean} aborted - true if any provider returned a result with the 'aborted' flag set
```

Comment:
Still need to update annotation. Also should update description about updating icon. Something like:

```js
    /**
     * Update the title of the problem panel, the tooltip of the status bar indicator, and the icon of
     * the status bar indicator. The title and the tooltip will change based on the number of problems
     * reported and how many provider reported problems. The icon changes for warnings or errors.
     * 
     * @param {Number} numProblems - total number of problems across all providers
     * @param {Array.<{name:string, scanFileAsync:?function(string, string):!{$.Promise}, scanFile:?function(string, string):Object}>} providersReportingProblems - providers that reported problems
     * @param {boolean} hasError - true if at least 1 problem is an error
     * @param {boolean} aborted - true if any provider returned a result with the 'aborted' flag set
     */
```